spec/rspec/core/option_parser_spec.rb

```diff
@@ -214,6 +214,10 @@ def generate_help_text
           options = Parser.parse([option, 'spec/**/*.spec'])
           expect(options[:pattern]).to eq('spec/**/*.spec')
         end
+        it 'will combine multple patterns' do
```

Comment:
A blank line between examples would be nice for consistent formatting :).

Also, the `will` seems superfluous...why not `it 'combines multiple patterns'`?